src/System.Private.CoreLib/shared/System/Type.cs

```diff
@@ -202,7 +203,7 @@ public MethodInfo GetMethod(string name, int genericParameterCount, BindingFlags
             return GetMethodImpl(name, genericParameterCount, bindingAttr, binder, callConvention, types, modifiers);
         }
 
-        protected virtual MethodInfo GetMethodImpl(string name, int genericParameterCount, BindingFlags bindingAttr, Binder binder, CallingConventions callConvention, Type[] types, ParameterModifier[] modifiers) => throw new NotSupportedException();
+        protected virtual MethodInfo GetMethodImpl(string name, int genericParameterCount, BindingFlags bindingAttr, Binder? binder, CallingConventions callConvention, Type[]? types, ParameterModifier[]? modifiers) => throw new NotSupportedException();
```

Comment:
`GetNestedType` overrides can also return null if it doesn't find a match: https://source.dot.net/#System.Private.CoreLib/src/System/RtType.cs,2956